src/Formatter/Formatter.php

```diff
@@ -112,6 +111,13 @@ protected function getConfigurator()
         $configurator->rendering->engine = 'PHP';
         $configurator->rendering->engine->cacheDir = $this->cacheDir;
 
+        $configurator->enableJavaScript();
+        $configurator->javascript->exportMethods = ['preview'];
+
+        $minifier = $configurator->javascript->setMinifier('FirstAvailable');
+        $minifier->add('MatthiasMullieMinify');
+        $minifier->add('Noop');
```

Comment:
@JoshyPHP Do we need the `FirstAvailable` strategy if we only use `MatthiasMullieMinify`? In other words, can that one fail in a way that is caught by `FirstAvailable`?

Reply:
I wouldn't know off the top of my head but after reading the API doc I think you can either use `FirstAvailable` in conjunction with `Noop` or you can a single minifier with `$keepGoing = true`, which also mutes exceptions.

```php
$configurator->javascript->setMinifier('MatthiasMullieMinify')->keepGoing = true;
```

Reply:
@sijad Please do that then. :see_no_evil: